src/toasts/toasts.stories.js

```diff
@@ -0,0 +1,10 @@
+import { storiesOf } from "@storybook/html";
+import html from "./toasts.stories.html";
+import "./toasts.css";
+import ejs from "ejs";
+
+import { htmlStory } from "../story-helpers";
+
+storiesOf("OpenMRS Styleguide", module).add("Toasts", () => {
+  return htmlStory(ejs.render(html, { text: "This is a toast" }));
```

Comment:
You're not using the `text` inside of the template. Should be removed.

```suggestion
  return htmlStory(ejs.render(html));
```